_data/education.yml

```
@@ -406,7 +406,7 @@ websites:
       - phone
       - proprietary
       - hardware
```

```suggestion
      - hardware
      - u2f
```
Based on their [FAQ](https://get2fa.psu.edu/faqs/), Penn State also supports U2F 2FA through Duo.